pkg/rotator/rotator.go

```diff
@@ -45,14 +45,17 @@ const (
 
 var crLog = logf.Log.WithName("cert-rotation")
 
-var vwhGVK = schema.GroupVersionKind{Group: "admissionregistration.k8s.io", Version: "v1beta1", Kind: "ValidatingWebhookConfiguration"}
-
 var crdGVK = schema.GroupVersionKind{Group: "apiextensions.k8s.io", Version: "v1beta1", Kind: "CustomResourceDefinition"}
 
 var _ manager.Runnable = &CertRotator{}
 
 var restartOnSecretRefresh = false
 
+type WebhookInfo struct {
```

Comment:
I know this code is "lightly" commented but it might be nice to start improving this.

Reply:
Done